tests/wikidata/conftest.py

```diff
@@ -20,3 +21,18 @@ def __init__(self: WikidataAPIConnector) -> None:
 
     monkeypatch.setattr(WikidataAPIConnector, "__init__", __init__)
     return mocked_session
+
+
+@pytest.fixture(autouse=True)
+def set_wiki_api_url_for_integration_tests(
+    monkeypatch: MonkeyPatch,
+    is_integration_test: bool,  # noqa: FBT001
+) -> None:
+    """Set correct Wikidata API URL for integration tests."""
+    if is_integration_test:
+        settings = BaseSettings.get()
+        monkeypatch.setattr(
+            settings,
+            "wiki_api_url",
+            "https://www.wikidata.org/w/api.php",
```

Comment:
Why not set this as the Settings.wiki_api_url default?

Reply:
currently all settings are set to unit-test compatible defaults. and i dont want unit tests calling the actual wikidata api. we could flip that around and have production-compatible defaults, but that would be a bigger discussion and refactor